assets/queries/terraform/alicloud/rds_instance_tde_status_disable/test/negative1.tf

```
@@ -0,0 +1,14 @@
+resource "alicloud_db_instance" "default" {
+    engine = "MySQL"
+    engine_version = "5.6"
+    db_instance_class = "rds.mysql.t1.small"
+    db_instance_storage = "10"
+    tde_status = "Enable"
```

```suggestion
    tde_status = "Enabled"
```

Please, also update the other test samples.